drivers/flash/spi_nor.c

```diff
@@ -1106,6 +1210,13 @@ static int spi_nor_configure(const struct device *dev)
 #endif /* CONFIG_FLASH_PAGE_LAYOUT */
 #endif /* CONFIG_SPI_NOR_SFDP_MINIMAL */
 
+#if DT_INST_NODE_HAS_PROP(0, mxicy_high_perf_mode)
+	rc = mx25r_configure(dev);
```

Comment:
This is a bit confusing. I agree that this configuration should be done always for MX25R chips, not only when the high performance mode is enabled (as the property description says: "If not specified, flash will be configured for Ultra low power mode"), but this checking if the property does not make it clear what is going on here. It would be better to check the node's compatible property.

Reply:
Yes, it was a mistake on my end not to use the same condition as I use around the function declarations.